ministryofjustice/laa-apply-for-criminal-legal-aid
# Crimapp 111 client have dependants

## Description of change

Adds in the "does client have dependants" step into the income journey flow.

## Link to relevant ticket
[CRIMAPP-111](https://dsdmoj.atlassian.net/browse/CRIMAPP-111)

## Notes for reviewer
There doesn't appear to be error copy for this step, I think its fine to go in now and may then updating.

## Screenshots of changes (if applicable)

### After changes:
<img width="668" alt="Screenshot 2023-11-30 at 13 08 15" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/13377553/c4f828ad-906e-4082-a30b-1edb41b8ef61">

## How to manually test the feature
Start an application and go to `/steps/income/does_client_have_dependants`
Page should render as per designs
Try to submit without selecting, you should receive an error message
Then select either yes or no and submit - it will take you to the index page for now as this step is out of flow.





[CRIMAPP-111]: https://dsdmoj.atlassian.net/browse/CRIMAPP-111?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
